README.adoc

```diff
@@ -951,7 +951,9 @@ end
 
 === Context Descriptions
 
-`context` block descriptions should always start with 'when', 'with', or 'without' and be in the form of a sentence with proper grammar when composed with `it` block descriptions.
+When an `it` block description is composed with a `context` block description, it should result in a sentence with
+proper grammar. This typically means the `context` should start with a term such as 'when', 'with', or 'without', and
```

Comment:
After the conversion to AsciiDoc we have that unwritten practice to use ventilated prose, i.e. put one sentence per line and not to use hard wraps.